### ISSUE TITLE
Blank Inventory Tab for Driveway Culvert Conveyance

### ISSUE THREAD

**Issue by cstevenson1** [leeensminger/DelDOT-NPDES-Field-Tool#12]
The inventory tab is completely blank after selecting a conveyance type of driveway culvert in the location tab.

![image](https://cloud.githubusercontent.com/assets/16919898/12680770/8d773d1e-c679-11e5-8afc-9f47877bdca5.png)

**Comment by cstevenson1** [comment]
Due to the blank tab, I ultimately could not create a driveway culvert conveyance because I was unable to save due to the notification of required fields needing a value.

**Comment by kanglu666** [comment]
fixed, ready for testing in next release

**Comment by cstevenson1** [comment]
Inventory tab is missing fields (no place to enter pipe properties). Fields for Shape, Material, Width, Height, Pipe Number, and Debris need to be added.

![image](https://cloud.githubusercontent.com/assets/16919898/13862654/fd22fe20-ec6a-11e5-8046-d6145d6db834.png)

**Comment by kanglu666** [comment]
we do not have roadway culvert and driveway culvert tables in database. related issue: #64 , #122

**Comment by leeensminger** [comment]
See same comment for 122.  This should be doable with the current database structure.

**Comment by kanglu666** [comment]
updated,ready for testing in next release

**Comment by cstevenson1** [comment]
Inventory Tab does not save changes. I updated the properties as shown in the first image below and clicked save. I then viewed another conveyance and re-opened the driveway culvert conveyance that I had previously changed the properties for and noticed the properties had changed to the default values as shown in the second image below. Also, debris is still missing from the inventory form. We do not want to be required to create an inspection for a driveway culvert conveyance in order to fill in debris since these are not DelDOT owned pipes.

![image](https://cloud.githubusercontent.com/assets/16919898/15116867/0c834bc2-15d3-11e6-981e-7f5b147d2b5c.png)

![image](https://cloud.githubusercontent.com/assets/16919898/15116893/21a3a074-15d3-11e6-9aaa-9487797ef9a1.png)
